wfc3tools/wf3cte.py

```diff
@@ -105,12 +36,8 @@ def wf3cte(input, output=None, parallel=True, verbose=False, log_func=print):
         - a single filename (``iaa012wdq_raw.fits``)
         - a Python list of filenames
         - a partial filename with wildcards (``*raw.fits``)
-        - filename of an ASN table (``*asn.fits``)
```

Comment:
same question i had on the other pr, why are asns no longer valid input?

Reply:
For this component, wf3cte, ASNs were never actually accommodated by the interface, so the improper documentation has been cleaned up.